eppo/src/main/java/cloud/eppo/android/EppoClient.java

```diff
@@ -175,13 +175,13 @@ public Builder assignmentCache(IAssignmentCache assignmentCache) {
     public Builder initialConfiguration(byte[] initialFlagConfigResponse) {
       this.initialConfiguration =
           CompletableFuture.completedFuture(
-              Configuration.builder(initialFlagConfigResponse, true).build());
+              new Configuration.Builder(initialFlagConfigResponse).build());
```

Comment:
Nice! By omitting it will parse the flag response and then check the format field 🚀